src/components/appointments/CreateAppointment/CreateAppointment.stories.tsx

```diff
@@ -181,7 +181,11 @@ export const HappyFlow: Story = {
       expect(await canvas.findByText('Jouw product(en)')).toBeVisible();
       expect(await canvas.findByText('Paspoort aanvraag: 1x')).toBeVisible();
       expect(await canvas.findByText('Jouw afspraak')).toBeVisible();
-      expect(await canvas.findByText(`Open Gem, Amsterdam, ${tomorrow}, 07:00`)).toBeVisible();
+
+      // the variable `tomorrow` has leading zeros which is not the case in the shown result
```

Comment:
can't we update the format then to not have these leading zeros?

Reply:
I still want the locale switcher (en/nl) to work properly in SB, but it looks like `date-fns` might be doing something different than `react-intl` here :thinking: 

Maybe we can use the imperative API from react-intl instead of date-fns here?

Reply:
Unfortunately this (the above suggestion) doesn't work either..